test/parallel/test-file-write-stream3.js

```diff
@@ -185,7 +185,7 @@ const run_test_4 = common.mustCall(function() {
   const err = {
     code: 'ERR_OUT_OF_RANGE',
     message: 'The value of "start" is out of range. ' +
-             'It must be >= 0 and <= 2 ** 53 - 1. Received -5',
+             'It must be >= 0 && <= 9007199254740991. Received -5',
```

Comment:
This change may confuse the reader. But if you want to keep it, please add a one line comment mentioning what it is.

Reply:
Yes, I would like to add a note stating that `2 ** 53 - 1` equals `9007199254740991`.

Reply:
replace `Number.MAX_SAFE_INTEGER` with `9007199254740991`